woocommerce/woocommerce-android
# Add option update simulator UI

Closes: #7690


### Description

Add the UI for the option to update the Simulated card reader on the Developer Options Screen. 

### Testing instructions


On Debug Build:
1 - Click on the Menu
2 - Click on Settings
3 - Click on the Developer Options row
4 - Click on the Update Simulated Reader row
5 - click on one of the options and click out of the dialog to dismiss it. 
6 - Click on the Row again to make sure the option you select has been saved

### Images/gif

<img src="https://user-images.githubusercontent.com/30724184/201993138-192d3cb8-8488-4d9a-80b2-596d4eb36b69.gif" width="350"/>
